kfp/pipeline_generator/superpipeline/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+DEF_FILE=$1
```

Comment:
can you take examples of arguments processing from `run.sh` in the  single pipeline generator

Reply:
I changed the implementation to use jinja as in the single pipeline generator.